docs/integrations/pytorch_lightning.rst

```diff
@@ -3,719 +3,182 @@
 Neptune-PyTorch Lightning Integration
 =====================================
 
-|Run on Colab|
+.. warning::
+    This is ``LegacyLogger`` implementation that is under maintenance support only.
+    No new updates will be made to this logger.
+
+    Please visit
+    `integration docs <https://docs.neptune.ai/integrations-and-supported-tools/model-training/pytorch-lightning>`_
+    to learn about the latest, fully supported version.
 
 What will you get with this integration?
 ----------------------------------------
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research.
+With Neptune integration you can:
 
-|pytorch-lightning-tour|
-
-PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
-
-* see experiment as it is running,
-* log training, validation and testing metrics, and visualize them in Neptune UI,
-* log experiment parameters,
+* monitor model training live,
+* log training, validation, and testing metrics, and visualize them in the Neptune UI,
+* log hyperparameters,
 * monitor hardware usage,
-* log any additional metrics of your choice,
+* log any additional metrics,
 * log performance charts and images,
 * save model checkpoints.
 
-.. note::
+Installation
+------------
+Before you start, make sure that:
 
-    This integration is tested with ``pytorch-lightning==1.2.1``, and ``neptune-client==0.5.1``.
+* You have Python 3.6+ in your system,
+* You are already `registered user <https://neptune.ai/register>`_, so that you can log metadata to your `private projects <https://docs.neptune.ai/administration/workspace-project-and-user-management/projects>`_.
+* You have your ``NEPTUNE_API_TOKEN`` set to the environment variable: `check this docs page <https://docs.neptune.ai/getting-started/installation#authentication-neptune-api-token>`_.
 
-Where to start?
----------------
-To get started with this integration, follow the :ref:`Quickstart <quickstart>` below. You can also skip the basics and take a look at the :ref:`advanced options <advanced-options>`.
+**Install neptune-client\[pytorch-lightning\]**
```

Comment:
Should be Install neptune-contrib[monitoring]

Reply:
fixed